app/backend/src/couchers/servicers/communities.py

```diff
@@ -180,6 +180,43 @@ def ListAdmins(self, request, context):
                 next_page_token=str(admins[-1].id) if len(admins) > page_size else None,
             )
 
+    def AddAdmin(self, request, context):
```

Comment:
I think you should do the permission check using `can_moderate_node(session, context.user_id, node.id)` (where `node.id == request.community_id`, but you probably need to grab the node first.